src/components/Footer/Footer.jsx

```diff
@@ -326,14 +327,9 @@ const Footer = () => {
                 aria-label="Follow me on Github"
                 target="_blank"
               >
-                <i className="fa-brands fa-github"></i>
+                <FaGithub size="1.39rem"/>
```

Comment:
- Please add the styles to the `css` file
 - Please make sure the icons has a color of #E26959  ?

Reply:
### Adjustments made--
I have changed the icon colour to #E26959.

### Screenshot--
![image](https://github.com/MilanCommunity/Milan/assets/89240074/6b0cb28b-b0dd-4197-a7f5-e30c82176a4d)

Reply:
That looks great, please make sure to remove any in-line stylings, and put all the stylings for color and size in the `.css` file. Once done, you can mark this as resolved and request for review. I'll merge it ASAP.
Great work.

Reply:
I have made the necessary changes in the file.